src/student/services/StoreService.tsx

```diff
@@ -1,40 +1,103 @@
-import { createContext, ReactNode, useState } from "react";
+import { createContext, ReactNode, useEffect, useState } from "react";
 
 export interface StoreProps {
     children: ReactNode
 }
 
 export interface IStore {
-    invitation?: string,
-    studentNick?: string,
+    invitation: string,
+    studentNick: string,
+    studentId: string,
     quizes: Quiz[],
 }
 
-const Store = (props: StoreProps) => {
-    const [invitation, setInvitation] = useState("");
-    const [studentNick, setStudentNick] = useState("");
-    const [quizes, setQuizes] = useState<Quiz[]>([]);
+enum StorageKey {
+    invitation,
+    studentNick,
+    studentId,
+    quizes
+}
+
+const stringKey = (key: StorageKey) => {
+    return "lazare.student." + key;
```

Comment:
why do you use "lazare..." annotation? is it only for easter-egg-like features or it does have a deeper meaning?

Reply:
Just convention to name keys in storage.
It was my personal choice(not some standard etc), but I think it looks cool xD